week2/index.js

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs';
+const argumentArray = process.argv;
+const command = process.argv[2];
+const argument = process.argv[3];
+switch (command) {
+  case "help":
+  case undefined:
```

Comment:
Instead of checking for `case undefined`, you can execute this on `default:`. For example:

```javascript
switch(command) {
  // other commands here
  case 'help':
  default:
    // show help
}
```

Would this work?